### ISSUE TITLE
See History of a Proposal

### ISSUE THREAD

**Issue by parabolika66** [MyrionPhoenixmoon/scenarioo#10]
As a user I want to be able to see the history of a proposal so that I can see how a proposal has evolved.

**Comment by adiherzog** [comment]
If the user saves the proposal several times during one edit session, only one revision should be created.

**Comment by MyrionPhoenixmoon** [comment]
Definitely. This entire feature is low priority though, since the need can be basically fulfilled through new proposals. Simply name them apropriately.
We simply wanted to record the idea.

**Comment by adiherzog** [comment]
That's true, it's quite low prio. Maybe we can prioritize the issues a bit tomorrow.
